gadk/elements.py

```diff
@@ -362,6 +363,10 @@ def on(
             self._on["workflow_dispatch"] = workflow_dispatch
         elif "workflow_dispatch" in self._on:
             del self._on["workflow_dispatch"]
+        if schedule is not None:
+            self._on["schedule"] = {"cron": schedule}
+        elif "schedule" in self._on:
+            del self._on["schedule"]
```

Comment:
What is this needed for?

Reply:
I've no idea, I was just following the existing pattern. There's lots here I don't understand and would love to change but not in this context.